.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cairo_contracts"]
+	path = lib/cairo_contracts
+	url = https://github.com/OpenZeppelin/cairo-contracts
```

Comment:
Is there a need to focus the main branch?
If not we should try to target a tag as much as possible I think